EN/Moodles_EN.txt

```diff
@@ -73,18 +73,18 @@ Moodles_EN = {
 	Moodles_hasacold_desc_lvl3 = "60%^ cold strength<br>sneezes and coughs often<br>-20% speed<br>-20% healing",
 	Moodles_hasacold_desc_lvl4 = "75%^ cold strength<br>sneezes and coughs a lot<br>-35% speed<br>-35% healing",
 	-- Sickness
-	Moodles_sick_desc_lvl1 = "25%^ Incidence<br>foraging vision reduced<br>small body heat increase<br>might lose HP",
-	Moodles_sick_desc_lvl2 = "50%^ Incidence<br>less foraging vision<br>body heat increase<br>-1 carrying capacity<br>-10% healing<br>might lose HP",
-	Moodles_sick_desc_lvl3 = "75%^ Incidence<br>even less foraging vision<br>high body heat increase<br>-2 carrying capacity<br>-20% healing<br>might lose HP",
-	Moodles_sick_desc_lvl4 = "90%^ Incidence<br>worse foraging vision<br>higher body heat increase<br>-3 carrying capacity<br>might lose HP",
+	Moodles_sick_desc_lvl1 = "25%^ Incidence<br>foraging vision reduced<br>small body heat increase<br>might lose HP (poison, zombification)",
+	Moodles_sick_desc_lvl2 = "50%^ Incidence<br>less foraging vision<br>body heat increase<br>-1 carrying capacity<br>-10% healing<br>might lose HP (poison, zombification)",
+	Moodles_sick_desc_lvl3 = "75%^ Incidence<br>even less foraging vision<br>high body heat increase<br>-2 carrying capacity<br>-20% healing<br>might lose HP (poison, zombification)",
+	Moodles_sick_desc_lvl4 = "90%^ Incidence<br>worse foraging vision<br>higher body heat increase<br>-3 carrying capacity<br>loses HP",
```

Comment:
Unless this was changed, from what I remember, even at 100% I think from food poisoning? you didn't lose HP, which is why they say "might lose HP".

```suggestion
	Moodles_sick_desc_lvl4 = "90%^ Incidence<br>You are poisoned or zombifying<br>worse foraging vision<br>higher body heat increase<br>-3 carrying capacity<br>might lose HP",
```

Reply:
I'll revert this

Reply:
I believe it might have change because sickness 4 Doesn't remove/reduce health regeneration. Which makes no sense unless it triggers hp decreases I'll look into that

Reply:
e2b9f207cc1fae16300d04759e0db80bb3c268f3